.docs/user-guide/config.md

```diff
@@ -589,6 +589,7 @@ remote storage systems. Currently the following storage drivers are supported:
 [EBS](./storage-providers.md#aws-ebs) | ebs, ec2
 [EFS](./storage-providers.md#aws-efs) | efs
 [RBD](./storage-providers.md#ceph-rbd) | rbd
+[GCE](./storage-providers.md#gce) | gce
```

Comment:
Hi @codenrhoden,

This driver abbreviation still reflects the old `gce` and not the updated `gcepd`.